src/openrct2-ui/windows/Ride.cpp

```diff
@@ -6206,7 +6206,7 @@ static_assert(std::size(RatingNames) == 6);
                 return;
             }
 
-            price = std::clamp(price, 0.00_GBP, 20.00_GBP);
+            price = std::clamp(price, kRideMinPrice, kRideMaxPrice);
```

Comment:
It’s good that the window does a bounds check, but the action should do so as well, I’d say. Would you agree?

Reply:
How that you mention it, I think the action should fail outright if anything out of bounds is presented, indeed.